dts/arm/nordic/nrf54l20_cpuapp.dtsi

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <nordic/nrf54l20.dtsi>
+
+cpu: &cpuapp {};
+systick: &cpuapp_systick {};
+nvic: &cpuapp_nvic {};
+
+/ {
+	soc {
+		compatible = "simple-bus";
+		interrupt-parent = <&cpuapp_nvic>;
+		ranges;
+	};
+
+	psa_rng: psa-rng {
+		compatible = "zephyr,psa-crypto-rng";
+		status = "disabled";
```

Comment:
Is PSA RNG not functional at this time?
If yes, then would it make sense to temporarily add a PRNG node?
```
        prng: prng {
		compatible = "nordic,entropy-prng";
		status = "okay";
	};
```
Lack of the entropy node in the SoC DTSI makes it difficult to build many applications, because you basically need to make an application DTS overlays with the snippet I added everywhere.

If you go that route, you also need to add `SOC_SERIES_NRF54LX` to `depends` of  `FAKE_ENTROPY_NRF_PRNG` Kconfig and set the `zephyr,entropy`.